.github/workflows/deploy-dev.yml

```diff
@@ -13,6 +13,9 @@ on:
       ocr-version:
         description: 'Create a version for this OCR API image'
         required: true
+      backend-version:
+        description: 'Create a version for this Backend API image'
+        required: true
```

Comment:
these inputs will go bye-go-bye soon but we can do that after since its a bit out of scope.